tests/linearGaussian_snpe_test.py

```diff
@@ -416,7 +416,7 @@ def simulator(theta):
         else:
             return linear_gaussian(theta, -likelihood_shift, likelihood_cov)
 
-    net = utils.posterior_nn("maf", hidden_features=20)
+    net = utils.posterior_nn("maf", z_score_x="structured", hidden_features=20)
```

Comment:
@michaeldeistler I changed this one to use structured z-scoring, is this sufficient?

Reply:
hm the `sample_conditional` might not be the perfect place for that. Maybe use some other function that actually tests the posterior (not the conditional posterior)